packages/ripple-binary-codec/src/enums/bytes.ts

```diff
@@ -0,0 +1,75 @@
+import { BytesList, BinaryParser } from '../binary'
```

Comment:
Unfortunately it doesn't seem possible (took a crack at it, but it was really hard to put things back so the diff was clean, and also seemed like it would cause the circular import issues again). 

I explicitly highlighted the areas that are mostly moving code along with a link to the changed code so you can more directly compare what happened by alt-tabbing between the highlight and changed code. (Along with explicitly calling out the little changes I made during the transition)